execution/executor/tests/db_bootstrapper_test.rs

```diff
@@ -254,21 +247,22 @@ fn restore_state_to_db(
 
 #[test]
 fn test_pre_genesis() {
-    let (mut config, genesis_key) = config_builder::test_config();
+    let genesis = vm_genesis::test_genesis_change_set_and_validators(Some(1));
+    let genesis_key = &vm_genesis::GENESIS_KEYPAIR.0;
```

Comment:
the GENESIS_KEY is confusing, can we name it something like LIBRA_ROOT_FOR_TEST?

Reply:
hmm... @tzakian  and I were discussing some sort of refactoring here, the naming is bad, the testing configuration is bad... we should separate the testing utility from the underlying builder framework.